### PR TITLE
LPD-40798

### PR DESCRIPTION
**Pull request by rijema**
Related issue: https://liferay.atlassian.net/browse/LPD-40798

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
@marcelabc , the changes were done. Thank you!

**Comment by rijema** [comment]
Conflict solved.

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -772,6 +667,194 @@ public void testAddObjectEntry() throws Exception {
...
+					"emailAddressRequired", "hosea@liferay.com"
+				).put(
+					"externalReferenceCode", externalReferenceCode
+				).put(
+					"firstName", "Hosea"
+				).put(
+					"listTypeEntryKeyRequired", "listTypeEntryKey2"
+				).build()));
+
+		// Object entry with unique values in a company scoped object definition
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:856]: Rename:
// Object entry with unique values in the same company

Comment (marcelabc) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:856]: After this renaming, you will need to reorder the test :disappointed: 

```
// Object entry with localized values
// Object entry with the same external reference code in different sites
// Object entry with the same external reference code in the same company
// Object entry with unique values in the same company
// Object entry with unique values in different sites
// Object entry with unique values in the same site
```

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by rijema** [comment]
Update available here: https://github.com/liferay-bpm/liferay-portal/pull/3965

**Pull request closed without merging**